src/main/java/org/example/model/Response.java

```diff
@@ -1,19 +1,17 @@
 package org.example.model;
 
+import java.util.List;
+
 public class Response {
     private int code;
     private String body;
     private String contentType;
+    private String jspAttribute;
+    private List<JspAttribute> jspAttributes;
 
     public Response() {
     }
 
-    public Response(int code, String body, String contentType) {
```

Comment:
Why you have removed the all args constructor?

Reply:
because i didn't use it

Reply:
Ok, but by the convention the POJO class should have full args constructor.